test/test_train_imagenet.py

```diff
@@ -164,10 +166,13 @@ def test_loop_fn(model, loader, device, context):
     return correct / total_samples
 
   accuracy = 0.0
+  writer = SummaryWriter(log_dir='/tmp/imagenet_tensorboard')
```

Comment:
This should definitely be the default, and the `log_dir` must be a command line argument.

Reply:
Defaulted to FLAGS.logdir